test/system/030-run.bats

```diff
@@ -1298,10 +1298,18 @@ search               | $IMAGE           |
     run_podman run -d -q --syslog $IMAGE sleep infinity
     cid="$output"
 
+    # The CIRRUS_CHANGE_TITLE env variable should not affect the tests below:
+    unset CIRRUS_CHANGE_TITLE
```

Comment:
@giuseppe how did this variable affect the test?  I'm trying to understand it but failed to see a connection

Reply:
it contained the title of the PR "libpod: skip DBUS_SESSION_BUS_ADDRESS in conmon" so DBUS_SESSION_BUS_ADDRESS was still found in the environment variables

Reply:
if I need to repush, I'll also change the check for `DBUS_SESSION_BUS_ADDRESS=` instead of `DBUS_SESSION_BUS_ADDRESS`

Reply:
it passes now